src/rpc/rpcquorums.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2017 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "server.h"
+#include "validation.h"
+
+#include "llmq/quorums_debug.h"
+#include "llmq/quorums_dkgsession.h"
+
+void quorum_dkgstatus_help()
+{
+    throw std::runtime_error(
+            "quorum dkgstatus (proTxHash) (detail_level)\n"
+            "\nArguments:\n"
+            "1. \"proTxHash\"          (string, optional, default=0) ProTxHash of masternode to show status for.\n"
+            "                        If set to an empty string or 0, the local status is shown.\n"
```

Comment:
```suggestion
            "                        If set to an empty string or "0", the local status is shown.\n"
```

Btw, is it really needed to support "0" at all?

Reply:
Hmm, not sure why I started adding this in the first place. Removing support for "0" now. But we also have this in a few `protx` commands, should clean this up later as well (or even before the release?).

Reply:
Yep, I guess it would make to drop it everywhere if there is no special meaning to it. Or maybe there are some issues with using empty strings in rpc via python (sentinel)? @nmarley